app/build.gradle

```diff
@@ -20,11 +20,29 @@ android {
 }
 
 dependencies {
+
     compile fileTree(dir: 'libs', include: ['*.jar'])
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:26.+'
+
+    compile 'com.android.support:support-v4:26.0.0-alpha1'
```

Comment:
Widzę, że dorzuciłeś do projektowego `build.gradle` odpowiednie repozytorium. Teraz powinieneś mieć już dostęp do wersji `26.0.1`. Jeśli nadal jednak coś nie działa to daj znać.

Reply:
Przeszło, dziwne nie wiem czemu wcześniej nie przechodziło.